sgx-types/src/tcs.rs

```diff
@@ -45,6 +45,8 @@ pub struct Tcs {
     fslimit: u32,
     /// Size to become a new GS-limit (only 32-bit enclaves).
     gslimit: u32,
+    reserved0: [u64; 23],
```

Comment:
The "extra bytes" not being zeroed by the compiler were the bytes added to by `align(4096)` on `Tcs`? Is it expected to run into that problem? Should we always pad out these structs if there's an alignment requirement?

Reply:
Basically, when a type is `align(4096)` the size of the type will always be a multiple of 4096. But the compiler will only ensure the actual fields are initialized, not the extra padding. The extra padding being initialized only cares when we are going to *measure* the whole size. Because, in this case, the padding's contents is undefined, the measurement result will be unpredictable.